frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/movement-animator.ts

```diff
@@ -52,9 +50,7 @@ export class MovementAnimator<T extends LineString | Point> {
         endPosition: Coordinates<T>
     ) {
         const startTime = Date.now();
-        const featureGeometry = feature.getGeometry()!;
-        const startPosition =
-            featureGeometry.getCoordinates() as Coordinates<T>;
+        const startPosition = this.getCoordinates(feature);
         // Stop an ongoing movement animation
         this.stopMovementAnimation(feature as Feature<T>);
         // We don't have to animate this
```

Comment:
```suggestion
        this.stopMovementAnimation(feature);
```
Unhelpful comments and useless type cast

Reply:
Applied!